data/windows.yaml

```diff
@@ -1807,6 +1894,41 @@ urls:
 - 'https://blog.talosintelligence.com/2019/05/threat-roundup-0517-0524.html'
 - 'https://blogs.technet.microsoft.com/platforms_lync_cloud/2017/05/05/disabling-windows-10-action-center-notifications/'
 ---
+name: WindowsBootConfigurationSettings
+doc: |
+  Windows Boot Configuration Settings.
+
+  These registry keys are associated with the Windows Boot
+  Configuration Data. Malware can modify these keys to disable
+  recovery options like the ability to boot into safe mode.
+  One malware family known to modify these keys is Cerber, a
+  well-known ransomware.
+
+  Modifications are most easily performed via bcdedit.exe. The
```

Comment:
Modifications are most easily performed via bcdedit.exe. => 

bcdedit.exe can be used to modify the Windows Boot Configuration Settings.